index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ERNEST Portfolio Website</title>
+</head>
+<body>
+    <!-- defining the header and wraping it in divs and will contain the logo -->
+    <div class="main_container">
+        <div class="inside_container">
+            <header class="main_header">
+              <nav class="navbar">
+                <a href="#" class="my-portfolio-logo">
+                    <img src="./assets/MyLogo.png" alt="My logo"/>
+                </a>
+                <a>
+                    <img src="./assets/menu.png" alt="menu-icon"/>
+                </a>
+              </nav>
+            </header>
+    <!-- Defining the main section for the description of the developer -->
+            <main class="main_section">
+                <div class="container">
+                   <div class="welcome_note"> <h2 class="h2">I am Ernest Lasten. Glad to see you!</h2></div>
+                    <p class="main_content">
+                        I’m a software developer! I can help you build a product ,
+                        feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                    <div class="connect_links">
+                        <strong class="connect_text">LET'S CONNECT</strong>
+                          <ul class="links_list">
+                            <li class="media_links">
+                                <a href="#"><img src="./assets/twitterIcon.svg" alt="icon for twitter"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href="#"><img src="./assets/linkedInIcon.png" alt="icon for linkedIn"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href="#"><img src="./assets/mIcon.svg" alt="icon for Medium"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href="#"><img src="./assets/gitHubIcon.svg" alt="icon for GitHub"/></a>
+                            </li>
+                            <li class="media_links">
+                                <a href="#"><img src="./assets/peaceIcon.svg" alt="icon for M"/></a>
+                            </li>
+                          </ul>
+                    </div>
+                </div>
+            </main>
+        </div>
```

Comment:
- [ ] Kindly add the missing background image to your design
![Screen Shot 2022-04-30 at 08 38 44](https://user-images.githubusercontent.com/28307960/166094835-ee11b878-4dd6-488f-b13c-5742aa22a0de.png)
this is the current project
<img width="363" alt="Screen Shot 2022-04-30 at 08 40 07" src="https://user-images.githubusercontent.com/28307960/166094892-74e6aab3-b94a-4e74-85de-4626e4a676d6.png">